spec/plans/finish.fmf

```diff
@@ -15,14 +12,7 @@ example: |
 
 /shell:
     summary:
-        Perform finishing tasks using shell (bash) scripts
-    description:
-        Execute arbitrary shell commands to finish the testing.
-
-        Default shell options are applied to the script, see the
-        :ref:`/spec/tests/test` key specification for more
-        details.
-
+        Perform finishing tasks using shell (bash) scripts.
```

Comment:
I'm thinking about simplifying this section even a little bit more: Here we still mention `shell` and `ansible` plugins, which means we need to maintain the list, add new items when a new `finish` plugin is implemented.

What about just explaining what the `finish` step is about, including a single concise example for user to get a quick idea, and pointing to the `/plugins/finish` section to learn all details.

The example could quickly demonstrate both plugins, for example like this:

```yaml
finish:
  - how: shell
    script: upload-logs.sh
  - how: ansible
    playbook: playbooks/cleanup.yaml
```

Reply:
In addition, if I remove all of them, especially the `where` I get the following warning when compiling
```bash
guide.rst:920: WARNING: undefined label: '/spec/plans/finish/where'
```

Reply:
> @psss If I remove all the items then the `finish` step becomes a draft, this could be confusing for people reading the documentation... what do you think?

I didn't mean to remove everything, the `where` and `when` keys should definitely stay. At least for now. Plus there would be the overall description and example, so that section should not be empty, right?